docker-compose.dev.yml

```diff
@@ -0,0 +1,42 @@
+version: '3'
```

Comment:
Thanks for fixing!

I will show my idea below.

1. Fix docker-compose.dev.yml
```yaml
version: '3'
services:
  app:
    build:
      args:
        RAILS_ENV: ${RAILS_ENV:-development}
    volumes:
      - .:/app
    environment:
      - RAILS_ENV=${RAILS_ENV:-development}
      - BUCKY_DB_NAME=${BUCKY_DB_NAME:-bucky_development}
 ```

2. Add docker-compose up procedures for development in README.md
```md
COMPOSE_FILE=docker-compose.yml:docker-compose.dev.yml docker-compose up -d
or
docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d
```

Reply:
Maybe You should fix docker-compose.without_db.yml also.

Reply:
Thanks,  I fixed compose files and README!